admin/src/components/Processors/Processors.js

```diff
@@ -8,17 +8,22 @@ const Processors = ({
   processors,
   onCreateProcessorClick,
   onEditProcessorClick,
+  onDeleteProcessorClick,
   isLoading,
   children
 }) => {
-  function onCreateProcessorHandler(e) {
+  function onCreateProcessorHandler() {
     onCreateProcessorClick && onCreateProcessorClick();
```

Comment:
@Jakedavies had a suggestion to make the props required and not check if they exist – I fixed it in a later commit (see master)

Reply:
👍